src/Bot.ts

```diff
@@ -249,25 +249,26 @@ async function init (): Promise<void> {
       }
     })
 
-    const tickers = fs.readdirSync(join(__dirname, "tick")).filter(file => file.endsWith('.js'))
-    for (const file of tickers) {
-      const ticker = (await import(`./tick/${file}`)).default
-      // check if ticker is a valid ticker
-      if (!ticker || typeof ticker !== "function") {
-        console.error("Ticker file is not valid " + file)
-        continue
+    if (token !== undefined) {
```

Comment:
Verständnisfrage ohne es kritisieren zu wollen: Warum prüfst du nach dem Token?

Reply:
weil block hier ist so programmiert, dass wenn es keinen tocken gibt, das programm nicht einfach mit 1 beendet wird, sondern er bleibt in einer endlos schelife hängen.

eigentlich müsste man das fixen, aber das sah kompliziert aus. also frage ich einfach den token ab -> problem gelöst.